tests/spec/test_lib.rb

```diff
@@ -124,15 +125,15 @@ def file_type_unset(comm_st, cont)
     true
   end
 
-  def changed_since_should_find(com_st, sec, context, desc)
+  def changed_since(com_st, sec, cont, desc)
     `echo '#! /bin/bash' > #{valid_test}`
     `chmod +x #{valid_test}`
-    gitarro = "#{script} -r #{repo} -c #{context} -d #{desc} -g #{git_dir}" \
-                   " -t #{valid_test} -f #{ftype} -u #{url}" \
-                   " --changed_since #{sec}"
-    puts `ruby #{gitarro}`
-    return false if failed_status(com_st, context)
-    true
+    gitarro = "#{script} -r #{repo} -c #{cont} -d #{desc} -g #{git_dir}" \
+              " -t #{valid_test} -f #{ftype} -u #{url} --changed_since #{sec}"
+    stdout = Open3.capture3("ruby #{gitarro}")
```

Comment:
why we are using open3.capture3 for catching the stdout? or there is somethig i\m missing

using  the basic in ruby does already the job
``` ruby
stdout = `cmd`
```

Reply:
Will check. I didn't know that backticks means 'run this command'.

Why are we using then:
```puts `cmd````
Instead of just:
````cmd````

Reply:
we are using
`puts` only for debugging mode. We could just use
```ruby
out = `uptime`
puts outs
```